app/validators/geocoding_validator.rb

```diff
@@ -12,6 +12,8 @@ def validate_each(record, attribute, value)
       Geocoder.configure(Geocoder.config.merge(http_headers: { "Referer" => organization.host }))
       coordinates = Geocoder.coordinates(geocoder_value(value))
 
+      Rails.logger.info("Geocoded #{geocoder_value(value).inspect} for result #{coordinates.inspect}")
```

Comment:
ℹ️ I decided that we should have some sort of logging for network calls. Seems important to surface what it's trying to geocode and what the results are.